edu.cuny.hunter.mylyngit.core/src/edu/cuny/hunter/mylyngit/core/analysis/MylynGitPredictionProvider.java

```diff
@@ -110,9 +122,16 @@ public void setJavaProjects(IJavaProject[] javaProjects) {
 	/**
 	 * Bump DOI when there is a method change.
 	 */
-	public void bumpDOI(IMethod method) {
+	public void bumpDOI(IMethod method) {	
+		
+		if (this.enclosingMethods.contains(method)) {
+			// the scaling factor for enclosing method
+			this.scaling.set(Kind.EDIT, (float) 5.6);
```

Comment:
This number could be adjusted. I use 8*0.7, because I think 4*0.7, 2*0.7 could be too small. If this is too large, I could reduce.

Reply:
Don't we want a larger number here?